src/store.js

```diff
@@ -183,17 +195,16 @@ class AsyncStorageStore {
     await storage.saveAccessData(accessData);
   }
 
+  /* eslint-disable class-methods-use-this */
```

Comment:
Why not change `getStorage` to a `static` method?

Maybe rename it to `generateNewStorage` or something like that, also, to indicate it is not fetching anything that already exists, but rather creating something new and empty.

Reply:
Created a new `generateStorage` helper to remove this from the class methods

Reply:
Actually, other methods are dependent on this method as well, and it is better to have all of these as method on the `STORE` instead of separate helpers so I changed to have it as a method with the eslint skip comment again